assets/models/system/deformable_detr_twostage_refine_r50_16x2_50e_coco/description.md

```diff
@@ -1,9 +1,12 @@
+
 `deformable_detr_twostage_refine_r50_16x2_50e_coco` model is from <a href="https://github.com/open-mmlab/mmdetection/tree/v2.28.2" target="_blank">OpenMMLab's MMDetection library</a>. This model is reported to obtain <a href="https://github.com/open-mmlab/mmdetection/blob/e9cae2d0787cd5c2fc6165a6061f92fa09e48fb1/configs/deformable_detr/metafile.yml#L55" target="_blank">box AP of 46.8 for object-detection task on COCO dataset</a>. To understand the naming style used, please refer to <a href="https://mmdetection.readthedocs.io/en/v2.28.2/tutorials/config.html#config-name-style" target="_blank">MMDetection's Config Name Style</a>.
 
 DETR has been recently proposed to eliminate the need for many hand-designed components in object detection while demonstrating good performance. However, it suffers from slow convergence and limited feature spatial resolution, due to the limitation of Transformer attention modules in processing image feature maps. To mitigate these issues, we proposed Deformable DETR, whose attention modules only attend to a small set of key sampling points around a reference. Deformable DETR can achieve better performance than DETR (especially on small objects) with 10 times less training epochs. Extensive experiments on the COCO benchmark demonstrate the effectiveness of our approach.
 
 > The above abstract is from MMDetection website. Review the <a href="https://github.com/open-mmlab/mmdetection/tree/v2.28.2/configs/deformable_detr" target="_blank">original-model-card</a> to understand the data used to train the model, evaluation metrics, license, intended uses, limitations and bias before using the model.
 
+<b>Deprecation Warning</b>: This model version is only compatible with mmdet <=2.28 and is deprecated. We recommend to use this model instead <a hresp="https://ml.azure.com/models/mmd-3x-deformable-detr_refine_twostage_r50_16xb2-50e_coco/version/8/catalog/registry/azureml" target="_blank">mmd-3x-deformable-detr_refine_twostage_r50_16xb2-50e_coco</a>(this is the corresponding model compatible with mmdet >= 3.1.0). In our model catalog, the models prefixed with mmdet-3x are compatible with mmdet >= 3.1.0.
```

Comment:
If we specify the version, we will have to keep releasing new version of 2.x models, as and when we update 3.x models. My suggestion would be to mention the name of the 3.x model on the model catalog, and link the to catalog. Let the user search it.